connPFM/deconvolution/roi_pfm.py

```diff
@@ -0,0 +1,221 @@
+import datetime
+import getpass
+import os
+import socket
+import subprocess
+
+import nibabel as nib
+import numpy as np
+from nilearn.input_data import NiftiLabelsMasker
+
+import atlas_mod
+from cli import _get_parser
+from Scripts.hrf_matrix import HRFMatrix
+from Scripts.run_stability_lars_bcbl import run_stability_lars
```

Comment:
I believe you're missing `run_stability_lars_bcbl` and the functions it's calling.